source/administration/changelog.md

```diff
@@ -4,6 +4,198 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+- Redesigned channel switcher (CTRL/CMD+K)
+- Browse direct and group message channels (CTRL/CMD+SHIFT+K) and reply to the most recent message (SHIFT+UP) with new shortcuts.
+
+### Improvements
+
+#### Web User Interface
+- Enter key now confirms deletion on the screens to delete a custom emoji and delete a channel
+- Team and channel URLs now replace accented characters with their ASCII equivalents
+- Recent menitons and flagged posts icons in the header are now highlighted when they are active in the right-hand sidebar
+- Empty rows are now ignored in the Send Email Invite modal
+- Enter key now confirms leaving a team from the Leave Team modal
+- Profile popover now opens when clicking a username in mobile browser view
+- /join now allows switching to a private channel to which the user has access
+- Improved the formatting of Mattermost content when copying and pasting to other apps
+- Added the ability for users to view and modify their online status from their profile picture in the header
+
+#### Notifications
+- Added "@here" to the list of channel-wide mentions in Account Settings
+- Added a reminder when your Mattermost window is refreshed if a status override slash command in used to set yourself as /away or /offline 
+- Users will see a confimation dialog when attempting to use @all or @channel in a channel with over 10 users
+- Messages for others being added to a channel no longer trigger channels to be unread
+
+#### Administration
+- Added CLI tool for permanently deleting channels
+- Channel Admins can now delete users messages within their channel if permitted in the System Console.
+- Errors are now logged when failing to load config through the command line
+- Reduced unnecessary database reads and writes when bulk importing users
+
+#### System Console
+- System Console main dropdown menu now has links to the Admin Guide, Troubleshooting Forum, Commercial Support Page and the About Mattermost dialog
+- Added the ability to enable Legacy Signature (AWS Signature V2) with S3 compatible servers
+
+#### Authentication
+- Added a redirect to the appropriate team or channel if navigating to a Mattermost URL when logged out
+- Team Invite links now open in the current tab
+
+#### Performance
+- Upgraded GORP to support connection timeouts on MySQL and missing database columns on MySQL and Postgres
+
+#### Integrations
+- Posts from webhooks that are greater than 4000 characters are now broken into multiple posts
+
+#### Enterprise Edition
+- Added an announcement banner for Admins to make maintenance announcements across the entire system configurable in the System Console
+
+### Bug Fixes
+- Dragging and dropping a file onto the left-hand sidebar no longer navigates away from Mattermost to open the file in the browser
+- Textbox will no longer overlap the center pane message area as it expands when typing
+- Fixed an issue where statuses could get stuck online after quitting the desktop app or closing the browser window
```

Comment:
was there any follow-up on this post: https://pre-release.mattermost.com/core/pl/ospjkocq9jf6dn4uty4xpat7oc